helpers/Authoring.php

```diff
@@ -123,5 +112,49 @@ public static function addRequiredResources($sourceDirectory, $relativeSourceFil
 
         return $returnValue;
     }
-
+    
+    /**
+     * Remove invalid elements and attributes from QTI XML. 
+     * @param string $qti
+     * @return string sanitized XML
+     */
+    public static function sanitizeQtiXml($qti)
+    {
+        $doc = self::loadQtiXml($qti);
+        
+        $xpath = new \DOMXpath($doc);
+        $xpath->registerNamespace('qti', "http://www.imsglobal.org/xsd/imsqti_v2p1");
```

Comment:
ok, I see that the ns has been hard coded there :  https://github.com/oat-sa/extension-tao-itemqti/blob/master/model/qti/ParserFactory.php#L437